e2e/create_account.loggedout.spec.ts

```diff
@@ -0,0 +1,22 @@
+import { expect, test } from '@playwright/test'
+
+test.describe('Create Account', () => {
+  test('should show an alert when an invalid email address is used', async ({
+    page,
+  }) => {
+    await page.goto('/')
+    // Allow time for SWC compilation
+    await expect(
+      page.getByRole('heading', { name: 'Loading…' }),
+    ).not.toBeVisible(
+      { timeout: 2 * 60 * 1000 }, // ...wait 2 minutes if necessary
+    )
```

Comment:
Why this instead of `.slow()` as used above? If this is more reliable, maybe make this a reusable function that we can call at the beginning of each test suite

Reply:
`test.slow()` increases the [timeout of the test](https://playwright.dev/docs/next/test-timeouts#test-timeout), but I don't believe it extends the timeout of a particular expectation. So if SWC compilation took longer than the standard expectation timeout, the test would fail, even if the test timeout was extended. Instead, we can just increase the timeout of this assertion to account for SWC compilation. I'll break it out into a reusable function!